extensions/mssql/src/dashboard/serviceEndpoints.ts

```diff
@@ -8,46 +8,69 @@ import * as azdata from 'azdata';
 import * as nls from 'vscode-nls';
 const localize = nls.loadMessageBundle();
 
-import * as Utils from '../utils';
+import * as utils from '../utils';
+
+const mgmtProxyName = 'mgmtproxy';
+const grafanaEndpointName = 'metricsui';
+const grafanaDescription = localize('grafana', "Metrics Dashboard");
+const logsuiEndpointName = 'logsui';
```

Comment:
is kibana === logsui? If so, nit: let’s keep the variables endpointname/description in sync